01_Data/src/layers/sequelize/repository/Subscription.ts

```diff
@@ -3,10 +3,18 @@
 //
 // SPDX-License-Identifier: Apache 2.0
 
+import { Sequelize } from 'sequelize-typescript';
 import { SequelizeRepository, Subscription } from '..';
 import { ISubscriptionRepository } from '../../..';
+import { SystemConfig } from '@citrineos/base';
+import { Logger, ILogObj } from 'tslog';
+
+export class SequelizeSubscriptionRepository extends SequelizeRepository<Subscription> implements ISubscriptionRepository {
+
+  constructor(config: SystemConfig, logger?: Logger<ILogObj>, namespace = Subscription.MODEL_NAME, sequelizeInstance?: Sequelize) {
```

Comment:
instead of passing namespace around constructors cant we just set it on the class like so:
```
export class SequelizeSubscriptionRepository extends SequelizeRepository<Subscription> implements ISubscriptionRepository {
   namespace = Subscription.MODEL_NAME
...
}
```
and I think when the base will use `this.namespace` it should pick up the right value right? This way we don't have to pass this around and can just define it on the repo as a property which may be a little cleaner - thoughts?

Reply:
I changed it to be assigned in the super constructor as Zihe mentioned; I think I like that better than having it be a field though they both are better 👍